src/Microsoft.SqlTools.ServiceLayer/EditData/Contracts/EditColumnInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.EditData.Contracts
+{
+    /// <summary>
+    /// Metadata about a column in an editable result set
+    /// </summary>
+    public class EditColumnInfo
+    {
+        /// <summary>
+        /// The name of the column
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Whether or not the column can be edited
```

Comment:
Can you update the comment to give an example or two of why a column might not be editable?  (e.g. computed column, idk what else)